src/backend/crud/projects.ts

```diff
@@ -7,6 +7,7 @@ import type {
 } from 'src/Types';
 import type { Response } from '@backend/Types';
 import { getUser } from '@backend/auth';
+import type { ApiPostInviteUserToProject } from 'src/pages/api/invite-user-to-project';
```

Comment:
IMO it feels a bit odd to import a type from an Astro page/API route. Should we just move `ApiPostInviteUserToProject` to the main `Types.ts` file?